git-commit-signing/README.md

```diff
@@ -0,0 +1,23 @@
+---
+display_name: Git commit signing
+description: Configures Git to sign commits using your Coder SSH key
+icon: ../.icons/git.svg
+maintainer_github: phorcys420
+verified: false
+tags: [helper, git]
+---
+
+# git-commit-signing
+
+This module downloads your SSH key from Coder and uses it to sign commits with Git.
+It requires `jq` to be installed inside your workspace.
```

Comment:
Could mention that it requires `curl` as well. Sometimes only one of `curl` or `wget` is available.

Reply:
bf7f9d04163e9c085a7c57306e4198aa84019fb0